assets/js/pages/HostRelevantPatches/HostRelevantPatchesPage.jsx

```diff
@@ -26,29 +26,40 @@ function HostRelevantPatches({ hostName, onNavigate, patches }) {
 
   const [displayedPatches, setDisplayedPatches] = useState(patches);
 
-  const file =
-    window?.URL?.createObjectURL && displayedPatches?.length > 0
-      ? window.URL.createObjectURL(
-          new File(
-            [
-              Papa.unparse(
-                {
-                  fields: [
-                    'advisory_type',
-                    'advisory_name',
-                    'advisory_synopsis',
-                    'update_date',
-                  ],
-                  data: displayedPatches,
-                },
-                { header: true }
-              ),
-            ],
-            `${hostName}-patches.csv`,
-            { type: 'text/csv' }
+  const [csvURL, setCsvURL] = useState(null);
+
+  useEffect(() => {
+    setCsvURL(
+      patches.length > 0
+        ? URL.createObjectURL?.(
+            new File(
+              [
+                Papa.unparse(
+                  {
+                    fields: [
+                      'advisory_type',
+                      'advisory_name',
+                      'advisory_synopsis',
+                      'update_date',
+                    ],
+                    data: patches,
+                  },
+                  { header: true }
+                ),
+              ],
+              `${hostName}-patches.csv`,
```

Comment:
nitpick: maybe we could put `${hostName}-patches.csv` in a constant, as it is used twice, so we don't mess up if we change in this at some point